src/hooks/core/websocket/useSimpleWebsocketHooks.tsx

```diff
@@ -61,7 +61,7 @@ const useSimpleWebsocket = (url, protocol = undefined) => {
     socket.onmessage = (event: MessageEvent) => {
       try {
         const json = JSON.parse(event.data);
-        if (json.message.message_type === 'text') {
+        if (json?.message?.message_type === 'text') {
           json.message.attachments = [];
         }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of optional chaining (`?.`) here is a good practice to prevent potential `undefined` or `null` errors when accessing nested properties. However, it's important to note that if `json.message` is `undefined`, the code will not throw an error but `json.message.attachments = [];` will fail silently. Consider adding a null check for `json.message` before assigning to its `attachments` property.
```diff
-        if (json?.message?.message_type === 'text') {
+        if (json?.message && json.message.message_type === 'text') {
           json.message.attachments = [];
         }
```
This way, you ensure that `json.message` exists before trying to assign to its `attachments` property.


<!-- This is an auto-generated comment by OSS CodeRabbit -->